greenCan/moderation/views.py

```diff
@@ -25,7 +26,6 @@ def index(request):
 def review_post(request, id):
     id = int(id)
     template_name = "moderation/templates/review-post.html"
-    print(request.method)
     if request.method == "POST":
```

Comment:
Can you please add tests to this function before merging it? So that it doesn't drop our coverage?